app/controllers/catalog_controller.rb

```diff
@@ -163,6 +163,11 @@ class CatalogController < ApplicationController
       pc.add_show_field 'publisher_label_tsim', label: 'Publisher'
     end
 
+    config.for_display_type 'Grant' do |pc|
+      pc.add_show_field 'pi', label: 'Primary Investigator', accessor: :linked_pi
```

Comment:
I believe the label should be `Principal Investigator`.

Reply:
Fixed.